modules/edu.gemini.seqexec.server/build.sbt

```diff
@@ -0,0 +1,20 @@
+import Settings._
+import Settings.Libraries._
+import Settings.LibrariesJVM._
+
+name := "edu.gemini.seqexec.server"
+
+libraryDependencies ++= Seq(
+  ScalaZCore,
+  ScalaZConcurrent,
+  "io.argonaut"        %% "argonaut"                  % "6.1",
+  "commons-httpclient" % "commons-httpclient"         % "2.0",
+  // OCS bundles
+  SpModelCore,
+  POT,
+  EpicsACM
+)
+
+unmanagedJars in Compile ++= Seq(
+  new File((baseDirectory in ThisBuild).value, "lib/squants_2.11-0.6.1.jar")
```

Comment:
I don't have a strong preference on this but my inclination is to leave it in the lib directory until squants has been updated and it can go into the managed dependencies.  Otherwise we have another thing that we have to depend on being up and running and reachable in order to build.  I mean, in general in the future.  For now obviously you'll be pulling in OCS dependencies from artifactory anyway.  What do you think?